contracts/test/MerkleGrantRoundPayout.test.ts

```diff
@@ -0,0 +1,264 @@
+// --- External imports ---
+import { artifacts, ethers, waffle } from 'hardhat';
+import { deployMockContract, MockContract } from 'ethereum-waffle';
+
+// --- Our imports ---
+import { MerkleGrantRoundPayout } from '../typechain/MerkleGrantRoundPayout';
+import { expect } from 'chai';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { Artifact } from 'hardhat/types';
+import { balanceOf, setBalance, tokens } from './utils';
+import { BalanceTree } from '../../utils/src/merkle-distributor/balance-tree';
+import { BigNumber, Contract } from 'ethers';
+
+// --- Parse and define helpers ---
+const { deployContract } = waffle;
+const { isAddress } = ethers.utils;
+const randomAddress = () => ethers.Wallet.createRandom().address;
+
+type Claim = {
+  index: number;
+  payee: string;
+  amount: number;
+  merkleProof: string[];
+};
```

Comment:
Let's move this to `@dgrants/types/src/grant.d.ts` and import it from there, this way it can be reused by the frontend